index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
 
```

Comment:
I suggest we choose colors that are easier to see and visually appealing at the same time to create an overall good user experience!